data/remnant/remnant 2 side missions.txt

```diff
@@ -0,0 +1,172 @@
+			`	Moments later your ship pings a spot on the landscape ahead indicating a clearing on a low hill. Nodding your assent, the <ship> swoops to a hover just above it. A glance at your cargo hold display shows the Remnant vanishing out the back lock.  Your exterior cameras show the equipment settling to the ground on repulsor pallets while the Remnant drop to the ground on what appear to be repulsor jets. A final message appears on your console.`
+			`	"Drop success. Get clear of the arrival zone and pick us up after the dust settles."`
+			`	The <ship> veers and accelerates away from the approaching wreck without waiting for your approval and retreats to a safe distance.`
+
+
+
+mission "Remnant: Shattered Light 3b"
+	name "Remnant: Ghost Touch"
+	description "Help the Remnant experiment on the Shattered Light."
+	source "Viminal"
+	destination "Viminal"
+	waypoint "Coluber"
+	to offer
+		has "Remnant: Shattered Light 2: done"
+	to complete
+		has "Remnant: Shattered Light 3: done"
+
+
+
+mission "Remnant: Shattered Light 4"
+	name "Shattered Light"
+	description "Escort the wreck back to Viminal."
+	landing
+	source "Far Monad"
+	destination "Viminal"
+	to offer
+		has "Remnant: Shattered Light 3: done"
+	on offer
+		fail "Remnant: Shattered Light 3b"
+		conversation
+			`Eventually you find a good vantage point several kilometers away where you can watch the approaching ship. The Shattered Light falls through the sky with deceptive slowness, made all the worse for the fact that you can see clouds and sky passing through it. As it nears the surface you see a ripple in the air lance upwards, followed by a concussive shockwave as the Shattered Light suddenly becomes very solid. Its engines flare to life with a roar that rattles your bones and the ship lurches to a stop in mid-air, before crunching down into the forest.`
+			`	The dust and debris settles within a few minutes, and soon you and the <ship> are skimming over the trees. The approach gives you ample time to appreciate just how massive the Shattered Light is. Literally everything that one could need to lay the foundations for a new colony, packed into a single ship. You note large gashes in the hull, along with massive sensor arrays and a shattered dome on top. Based on the mismatch of parts, this ship has been substantially modified from its original hull, and survived untold damage.`
+			`	As you slow to a hover alongside, Ruach appears on your viewscreen. "Well met, Captain! Our theory worked! We managed to snap the Shattered Light back into the primary weft, and are now onboard. The systems are dubious, though, so we dare not shut anything down lest we cannot restart it. We would appreciate your assistance in escorting us back to Viminal."`
+				accept
+	npc accompany
+		government "Remnant (Research)"
+		personality launching timid escort
+		planet "Far Monad"
+		ship "Dromedary Wreck" "Shattered Light"
```

Comment:
When talking with this ship it said that I could not understand what they said (remnant pre understanding their language), is it just because I did the translation missions before the addition of the Remnant(Research) government? Shouldn't there be a patch for that?

Reply:
Yes, there should be. I'm not sure there is, though. I'll have to look into this.